### PR TITLE
Enable for other products

### PR DESCRIPTION
**Pull request by kskrygan**
Enable Azure plugin for other IJ-based products.

### CODE REVIEW

**Comment by vscjavaci** [comment]
Can one of the admins verify this patch?

File: PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -23,6 +23,7 @@
   <!-- uncomment to enable plugin in all products
   <depends>com.intellij.modules.lang</depends>
   -->
+  <depends>com.intellij.modules.platform</depends>
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml]: What product will this change support? We should have a good knowledge of the products and do the test, before we announce support

**Comment by adashen** [comment]
retest this please

File: PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml (hunk 1 above)
Comment (kskrygan) [inline PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml]: Specifically we need that for Rider - new cross platform .Net IDE based on top of Intellij.
jetbrains.com/rider

**Comment by adashen** [comment]
Not decided whether to support rider. Close the pull request for now

**Pull request closed without merging**